presto-docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -158,9 +158,9 @@ This property is optional; no default value.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Limits the number of elements returned in one batch. A cursor typically fetches a batch of result objects and stores them locally.
-If batchSize is 0, Driver's default will be used.
+If batchSize is 0, Driver's default are used.
 If batchSize is positive, it represents the size of each batch of objects retrieved. It can be adjusted to optimize performance and limit data transfer.
-If batchSize is negative, it will limit of number objects returned, that fit within the max batch size limit (usually 4MB), and cursor will be closed. For example if batchSize is -10, then the server will return a maximum of 10 documents and as many as can fit in 4MB, then close the cursor.
+If batchSize is negative, it limits the number of objects returned, that fit within the max batch size limit (usually 4MB), and the cursor is closed. For example if batchSize is -10, then the server returns a maximum of 10 documents,and as many as can fit in 4MB, then closes the cursor.
```

Comment:
Missing space after comma (`documents,and`)

Reply:
Fixed .. btw. what sort of line length do you prefer as standard (if there is any). The mixture of hard and soft wraps makes the source trickier to work with than necessary.